amy/consents/tests/test_action_required_view.py

```diff
@@ -1,8 +1,9 @@
+from amy.consents.tests.helpers import reconsent
```

Comment:
We don't import `from amy`, if you need you can import relatively `from .helpers` or from package `from consents.tests.helpers`.

Reply:
That should be resolved now